dallinger/command_line.py

```diff
@@ -853,14 +853,19 @@ def verify_package(verbose=True):
     """Ensure the package has a config file and a valid experiment file."""
     is_passing = True
 
-    # Check the config file.
-    if os.path.exists("config.txt"):
-        log("✓ config.txt is OK",
-            delay=0, chevrons=False, verbose=verbose)
-    else:
-        log("✗ config.txt is MISSING",
-            delay=0, chevrons=False, verbose=verbose)
-        return False
+    # Check for existence of required files.
+    required_files = [
+        "config.txt",
+        "experiment.py",
+        "requirements.txt",
+    ]
+
+    for f in required_files:
+        if os.path.exists(f):
+            log("✓ {} is OK".format(f), chevrons=False, verbose=verbose)
```

Comment:
can we change OK to PRESENT? Having OK gives the impression that we are somehow checking the contents of the files, when we are actually just checking that they exist